docs/build-on-linea/gas-fees.md

```diff
@@ -14,9 +14,9 @@ sidebar_position: 6
 
  2. We don't mine a transaction if `gasPrice` or `maxPriorityFeePerGas` is lower than a given value that fluctuates over time.
 
-To ensure that your transaction gets included by the sequencer, we recommend using EIP-1559 with the following settings:
+To ensure that your transaction gets included by the sequencer, we recommend using EIP-1559 with the following settings for non-MetaMask users:
```

Comment:
```suggestion
To ensure that your transaction gets included by the sequencer, we recommend that non-MetaMask users use EIP-1559 with the following settings:
```

Reply:
"users use" is not ideal but I think the order of the sentence does need to change to improve flow. I tried a few other options but this was the one I was most comfortable with